wsd/Storage.hpp

```diff
@@ -627,6 +627,14 @@ class WopiStorage : public StorageBase
     /// Total time taken for making WOPI calls during uploading.
     std::chrono::milliseconds getWopiSaveDuration() const { return _wopiSaveDuration; }
 
+    virtual AsyncUpload queryLocalFileToStorageAsyncUploadState() override
```

Comment:
I thought I had this function! Apparently I only had it in StorageBase, but never overridden it. Thanks for adding it.